### ISSUE TITLE
Notey Button

### ISSUE THREAD

**Issue by ghost** [GriefMoDz/notey#2]
When I click the button at the top right is says
"Something went wrong! Check your DevTools Console.." yknow the error message-
The consoles says this
```
TypeError: Cannot read property 'onKeyDown' of undefined
    at E (be76dfc695170334526d.js:3868)
    at ra (be76dfc695170334526d.js:4220)
    at Ks (be76dfc695170334526d.js:4272)
    at Lu (be76dfc695170334526d.js:4263)
    at bu (be76dfc695170334526d.js:4263)
    at Ru (be76dfc695170334526d.js:4263)
    at mu (be76dfc695170334526d.js:4260)
    at be76dfc695170334526d.js:4205
    at t.unstable_runWithPriority (be76dfc695170334526d.js:4289)
    at Hi (be76dfc695170334526d.js:4205)
    at ji (be76dfc695170334526d.js:4205)
    at Yi (be76dfc695170334526d.js:4205)
    at Me (be76dfc695170334526d.js:4282)
    at Zt (be76dfc695170334526d.js:4182)
    at HTMLDivElement.listener (<anonymous>:35:14)
(anonymous) @ f85482b23437d81b5c54.js:138
```

**Comment by GriefMoDz** [comment]
Thanks for the report! Are you able to confirm that this has been fixed as of https://github.com/GriefMoDz/notey/commit/6931069eda411fbe11f377a4f754684cba218441?

**Comment by ghost** [comment]
It has been fixed with 6931069.
